.github/linters/.arm-ttk.psd1

```diff
@@ -4,11 +4,13 @@
 @{
     # Test = @( )
     Skip = @(
-        'Template Should Not Contain Blanks',
-        'DeploymentTemplate Must Not Contain Hardcoded Uri',
-        'DependsOn Best Practices',
-        'Outputs Must Not Contain Secrets',
-        'IDs Should Be Derived From ResourceIDs',
+        'Template Should Not Contain Blanks'
```

Comment:
Please replace "should" with "must". 

Thanks

Reply:
This is a predefined rule. We cannot change this or functionality of the linter would change.